simplq/src/store/appSlice.js

```diff
@@ -14,7 +14,7 @@ const appSlice = createSlice({
   initialState: {
     errorText: '',
     notificationPermission: getNotificationStatus(),
-    loggedInUser: null,
+    isLoggedIn: false,
```

Comment:
We can't set this object in the store, as it contains functions inside it, which mutates itself, an redux doesn't like that.